test_common/harness/testHarness.h

```diff
@@ -72,13 +72,17 @@ extern cl_uint gRandomSeed;
 
 // Supply a list of functions to test here. This will allocate a CL device, create a context, all that
 // setup work, and then call each function in turn as dictatated by the passed arguments.
+// Returns EXIT_SUCCESS iff all tests succeeded or the tests were listed,
+// otherwise return EXIT_FAILURE.
 extern int runTestHarness( int argc, const char *argv[], int testNum, test_definition testList[],
                            int imageSupportRequired, int forceNoContextCreation, cl_command_queue_properties queueProps );
 
 // Device checking function. See runTestHarnessWithCheck. If this function returns anything other than TEST_PASS, the harness exits.
 typedef test_status (*DeviceCheckFn)( cl_device_id device );
 
 // Same as runTestHarness, but also supplies a function that checks the created device for required functionality.
+// Returns EXIT_SUCCESS iff all tests succeeded or the tests were listed,
```

Comment:
Spelling mistake `iff` -> should be `if`

Reply:
@TijmenUU, probably it is intended, see [if and only if](https://en.wikipedia.org/wiki/If_and_only_if)

Reply:
Thanks, I deleted my other comments, leaving this one for the author to review and possibly dismiss _iff_ I was indeed mistaken :wink:

Reply:
Yes, it was intended to mean _if and only if_. :wink: